### PR TITLE
Feat badge

### PR DESCRIPTION
**Pull request by abhij1607**
image
Feat: Badge

### CODE REVIEW

File: components/badge/badge.css [hunk 1]
```diff
@@ -0,0 +1,23 @@
+.badge {
+  width: 1.5rem;
+  height: 1.5rem;
+  border-radius: 50%;
+  position: absolute;
+  right: 10%;
+}
+.badge-online {
+  background-color: green;
```
Comment (AnuragSahu11) [inline components/badge/badge.css]: Try to use variables as much as possible, especially with colors

File: components/badge/badge.html [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+            display: flex;
+            justify-content: center;
+        }
+        .badge-box i{font-size: 3.2rem;}
+
+    </style>
+</head>
+<body>
+    <div class="wrapper">
+        <div class="container-bdage-avatar">
```
Comment (AnuragSahu11) [inline components/badge/badge.html]: container-bdage-avatar ,  there is a typo i think

**Review by AnuragSahu11 — COMMENTED** [review]
Other than that looks good

**Review by omjadhav85 — COMMENTED** [review]
Other than Anurag's suggestions, try adding a solid white border around the avatar badges...will look better I guess. Other that that looks clean!

File: components/badge/badge.css [hunk 3]
```diff
@@ -0,0 +1,23 @@
+.badge {
+  width: 1.5rem;
+  height: 1.5rem;
+  border-radius: 50%;
+  position: absolute;
```
Comment (lannisterrr) [inline components/badge/badge.css:5]: suggestion would be to add the badge at bottom right . With some border around.

Comment (abhij1607) [reply components/badge/badge.css:5]: thanks will sure apply

File: components/badge/badge.css (hunk 1 above)
Comment (abhij1607) [reply components/badge/badge.css]: Sure I'll change absolute colors to variables

File: components/badge/badge.html (hunk 2 above)
Comment (abhij1607) [reply components/badge/badge.html]: thanks for pointing it out

**Pull request merged**